### PR TITLE
Add a feedback button

### PR DESCRIPTION
**Pull request by OceanOak**
Changelog:

```
Internal improvements
- Add a new feedback button that points to github discussions.
- Add a new report a bug button that points to github issues.
- Convert the account actions menu to Tailwind CSS.
```
This solves  #4558, adds a report bug button, and converts the account actions menu to Tailwind.

Screenshot (165)

Still not sure about the Post feedback icon I will change it if I find something better.

### CODE REVIEW

**Comment by OceanOak** [comment]
I wanted to try adding Icons to the menu (github icon and discord icon), but I couldn't since they are not included in `https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css`. So, I tried to add `https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/brands.min.css` to be able to use them.

In `ui.html` I added:
```
<link rel="preload" as="style" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/brands.min.css" integrity="sha512G/T7HQJXSeNV7mKMXeJKlYNJ0jrs8RsWzYG7rVACye+qrcUhEAYKYzaa+VFy6eFzM2+/JT1Q+eqBbZFSHmJQew==" crossorigin="anonymous" referrerpolicy="no-referrer"/>
```

and 

```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/brands.min.css" integrity="sha512-G/T7HQJXSeNV7mKMXeJKlYNJ0jrs8RsWzYG7rVACye+qrcUhEAYKYzaa+VFy6eFzM2+/JT1Q+eqBbZFSHmJQew==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

The http://darklang.localhost:9000/a/dark stopped loading and I got the error bellow in the devtools console. Is there something I am doing wrong or am I missing some steps?

Screenshot (97)

**Comment by pbiggar** [comment]
Looks like a typo somewhere and it can't read the HTML or embedded JS.

**Comment by OceanOak** [comment]
> Looks like a typo somewhere and it can't read the HTML or embedded JS.
> Alternatively, you might need to ensure the versions match?

The only thing that fixed it is disabling the 'format on save' feature in VSCode. (the version didn't matter but I matched it anyway) 


These are possible options of **text-direction** and **icons**. Any preference?

account-actions

**Comment by StachuDotNet** [comment]
I like the option on the right - it seems more natural, since the icon that opens the menu is 'on the right'

**Comment by pbiggar** [comment]
I like the right also.

I'd suggest using the GitHub icon for "Report bug" as that will be clearer. For post feedback, could use a different icon such as one of the "writing an article" icons.

File: client/src/canvas/View.res [hunk 1]
```diff
@@ -634,33 +647,56 @@ let accountView = (m: model): Html.html<msg> => {
...
   )
 
-  let share = Html.p(
+  let reportBugRef = Html.a(
+    list{
+      tw(accountActionBtnStyle),
+      Attrs.href("https://github.com/darklang/dark/issues"),
+      Attrs.target("_blank"),
+      EventListeners.eventNoPropagation(~key="github-issues-ref", "click", _ => Msg.UpdateHeapio(
+        OpenKeyboardRef,
```
Comment (pbiggar) [inline client/src/canvas/View.res:675]: The OpenKeyboardRef listener isn't the right one - but you can just remove this an use no listener.

File: client/src/canvas/View.res [hunk 2]
```diff
@@ -634,33 +647,56 @@ let accountView = (m: model): Html.html<msg> => {
...
+  let feedbackRef = Html.a(
     list{
-      Attrs.class("account-action-btn invite"),
-      EventListeners.eventNoPropagation(~key="open-invite", "click", _ => Msg.SettingsMsg(
-        Open(Invite),
+      tw2(accountActionBtnStyle, %twc("mr-1")),
+      Attrs.href("https://github.com/darklang/dark/discussions/categories/feedback"),
+      Attrs.target("_blank"),
+      EventListeners.eventNoPropagation(~key="github-feedback-ref", "click", _ => Msg.UpdateHeapio(
+        OpenKeyboardRef,
```
Comment (pbiggar) [inline client/src/canvas/View.res:691]: Same as above.

**Review by pbiggar — CHANGES_REQUESTED** [review]
Looks good, just two minor things to go.

File: client/src/canvas/View.res (hunk 1 above)
Comment (OceanOak) [reply client/src/canvas/View.res]: What would be the right listener if we wanted to have one?

Comment (pbiggar) [reply client/src/canvas/View.res]: You'd add a new value to the place where `OpenKeyboardRef` is defined

**Comment by pbiggar** [comment]
Feel free to add back the Event listeners, but nbd if you don't.

File: client/src/canvas/View.res (hunk 1 above)
Comment (OceanOak) [reply client/src/canvas/View.res]: Ok, Thanks.

**Pull request merged**